git-workflows/hooks/detect-git-workflow.sh

```
@@ -0,0 +1,156 @@
+    operation="branch creation"
+
+# Sync skill
+elif matches_pattern "${sync_patterns[@]}"; then
+    skill_name="syncing-branch"
+    skill_desc="branch synchronization with fork/origin detection"
+    operation="sync operations"
+
+# Commit skill (most common, check last to avoid false positives from "commit and PR")
+elif matches_pattern "${commit_patterns[@]}"; then
+    skill_name="creating-commit"
+    skill_desc="atomic commits with code review and validation"
```

Shellcheck (0.11.0)

[warning] skill_desc appears unused. Verify use (or export if used externally).

(SC2034)